main.go

```diff
@@ -115,7 +114,7 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
-	if err := logsv1.ValidateAndApply(logOptions, nil); err != nil {
+	if err := logOptions.ValidateAndApply(nil); err != nil {
```

Comment:
Is this right? I thought I made a mistake here and the fix was to use that logv1 package instead.

Reply:
Yes, logv1 package is in v0.25 and it was causing the bump.

Reply:
When that line is not removed, go modules pull k8s.io v0.25, you can see this locally.

Reply:
Ah, okay. :)